templates/object_preview.html

```diff
@@ -1,10 +1,10 @@
 {% load thumbnail %}
 
-<a class="artist-details name" href="{{ object.url }}">
-    {% thumbnail img "250x250" crop="center" as thumb %}
-        <img src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ object.name }}" />
-    {% endthumbnail %}
-    <div class="artist-details artist-name">
-        {{ object.name }}
-    </div>
-</a>
+<div data-content="{{ object.name }}" class="name">
```

Comment:
You're saying that it's hard to tell that the artist pages are clickable? My thought was to move the name to appear on hover instead of below the image. It felt a little more interested when looking through the list. I could be wrong though